docs/conf.py

```diff
@@ -46,6 +46,7 @@
     "sphinx.ext.mathjax",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
+    "sphinx.ext.napoleon",
```

Comment:
ahhhh... yes missing extension!! great catch . i wonder if this will fix @nkorinek issues as well?

Reply:
I am confused about why master was building without this (because the docstrings with headings predate this PR)